server/src/test/java/org/elasticsearch/indices/state/RareClusterStateIT.java

```diff
@@ -149,20 +149,15 @@ public void onFailure(String source, Exception e) {
 
     public void testDeleteCreateInOneBulk() throws Exception {
         internalCluster().startMasterOnlyNode(Settings.builder()
-            .put(TestZenDiscovery.USE_ZEN2.getKey(), false) // TODO: convert test to support Zen2
+            //short publish timeout is needed not to wait too long on operation ack during cluster disruption
+            .put(Coordinator.PUBLISH_TIMEOUT_SETTING.getKey(), "1s")
```

Comment:
this means that the test will now fail on a slow CI machine that takes longer than "1s" to commit the cluster state update. I think we need to find a different way of writing this test, not relying on the `publish_timeout`. The alternative is that we have to reintroduce commit_timeout to Zen2 as well, just for testing purposes :/